### ISSUE TITLE
Display issues

### ISSUE THREAD

**Issue by mikeNG** [LegacyXperia/local_manifests#2]
Display lags heavily.

logcat:

```
E/qdgralloc(  233): fb_post: FBIOPUT_VSCREENINFO ioctl failed, err: Invalid argument
E/qdhwcomposer(  233): hwc_set_primary: ctx->mFbDev->post fail!
E/qdoverlay(  233): Failed to call ioctl MSMFB_OVERLAY_PLAY err=Operation not permitted
E/qdoverlay(  233): MdpData failed to play
E/qdoverlay(  233): == Dump MdpData start ==
E/qdoverlay(  233): == Dump OvFD fd=36 path=/dev/graphics/fb0 start/end ==
E/qdoverlay(  233): mOvData msmfb_overlay_data id=2
E/qdoverlay(  233): data msmfb_data offset=0 memid=23 id=0 flags=0x0 priv=0
E/qdoverlay(  233): == Dump MdpData end ==
E/qdhwcomposer(  233): draw: queueBuffer failed for external
E/qdhwcomposer(  233): hwc_set_primary: FBUpdate::draw fail!
```

dmesg flood:

```
<3>[   15.635711] mdp4_overlay_mdp_perf_req mdp pipe bw request should not be zero!
<6>[   15.652984] mdp4_overlay_mdp_perf_upd mdp clk is changed [1] from 0 to 190000000
<3>[   15.844604] mdp4_overlay_play: pmem Error
```

**Comment by mikeNG** [comment]
The above are solved, but we still have lower FPS than cm-10.1. This needs investigation.

**Comment by avinashrocks1990** [comment]
lag has somewhat improved in alpha2 but still 
it lags too much
